app/views/steps/solicitor/contact_details/edit.html.erb

```diff
@@ -1,20 +1,20 @@
 <% title t('.page_title') %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
     <%= step_header %>
```

Comment:
step_header goes outside the grid. It is correct in the other file `personal_details/edit.html.erb`
This file also is missing the error summary `<%= govuk_error_summary %>` (the other file is correct)

Reply:
thank you, solved